js/latoken.js

```diff
@@ -0,0 +1,635 @@
+        return trades;
+    }
+
+    async fetchMyTrades (symbol = undefined, params = {}, limit = 10) {
+        await this.loadMarkets ();
+        let request = {
+            'symbol': symbol,
+            'limit': 1000000000,
+        };
+        let response = await this.privateGetOrderTrades (this.extend (request, params));
+        let pairId = response.pairId;
+        let sym = response.symbol;
+        let tradeCount = response.tradeCount;
+        let result = [];
+        for (let i = 0; i < response.trades.length; i++) {
+            let resp = response.trades[i];
+            result.push ({
+                'pairId': pairId,
+                'symbol': sym,
+                'tradeCount': tradeCount,
+                'id': resp.id,
+                'orderId': resp.orderId,
+                'commision': resp.commision,
+                'side': resp.side,
+                'price': resp.price,
+                'amount': resp.amount,
+                'timestamp': resp.time,
+            });
+        }
+        let trades = this.parseTrades (result);
+        return trades;
+    }
+
+    parseOrderStatus (status) {
+        let statuses = {
+            'active': 'open',
+            'partially_filled': 'open',
+            'filled': 'closed',
+            'canceled': 'canceled',
+        };
+        return (status in statuses) ? statuses[status] : status;
+    }
+
+    parseOrder (order) {
+        let symbol = order['symbol'];
+        let side = order['side'];
+        let orderType = order['orderType'];
+        let price = this.safeFloat (order, 'price');
+        let amount = this.safeFloat (order, 'amount');
+        let orderStatus = order['orderStatus'];
+        let executedAmount = this.safeFloat (order, 'executedAmount');
+        let reaminingAmount = this.safeFloat (order, 'reaminingAmount');
+        let timeCreated = this.safeValue (order, 'timeCreated');
+        return {
+            'id': order['orderId'],
+            'info': order,
+            'timestamp': timeCreated,
+            'datetime': this.iso8601 (timeCreated),
+            'lastTradeTimestamp': undefined,
+            'status': orderStatus,
+            'symbol': symbol,
+            'type': orderType,
+            'side': side,
+            'price': price,
+            'cost': undefined,
+            'amount': amount,
+            'filled': executedAmount,
+            'average': price,
+            'remaining': reaminingAmount,
+            'fee': undefined,
+        };
+    }
+
+    async fetchActiveOrders (symbol = undefined, since = undefined, limit = 50, params = {}) {
+        return this.fetchOrdersByStatus (symbol, 'active', since, limit, params);
+    }
+
+    async fetchCancelledOrders (symbol = undefined, since = undefined, limit = 50, params = {}) {
+        return this.fetchOrdersByStatus (symbol, 'cancelled', since, limit, params);
+    }
+
+    async fetchFilledOrders (symbol = undefined, since = undefined, limit = 50, params = {}) {
```

Comment:
This is called `fetchClosedOrders` according to the Manual and to https://github.com/ccxt/ccxt/wiki/Requirements

Reply:
done